DockerfileFull

```diff
@@ -1,24 +1,25 @@
-FROM nginx:mainline-alpine
+FROM node:16 as builder
 
 ARG VUE_APP_G5V_API_URL
-RUN touch .env.production
-RUN echo $VUE_APP_G5V_API_URL < .env.production
-
-# install prerequisits 
-RUN apk update && apk add \
-	nodejs \
-	yarn	
-
-# Move default.conf into /etc/nginx/conf.d/
-COPY default.conf /etc/nginx/conf.d/default.conf
+ENV VUE_APP_G5V_API_URL=$VUE_APP_G5V_API_URL
 
 # create folder and copy application into folder
-RUN mkdir G5V
-COPY ./ /G5V/
 WORKDIR /G5V
+COPY package*.json .
+COPY yarn.lock .
 
 # build application using yarn
-RUN yarn && yarn build
+RUN yarn
+
+COPY . .
+
+ENV NODE_ENV=production
 
-# copy results to nginx directory
-RUN cp -R ./dist/* /usr/share/nginx/html
+RUN yarn build
+
+
+FROM nginx:latest
```

Comment:
The only change I would like to request is swapping this back to mainline-alpine instead of latest. I'd much prefer to have a smaller image footprint than have the entire release :)

Reply:
Updated :)